test/unit/module.spec.js

```diff
@@ -25,7 +25,7 @@ describe('application modules', function() {
 
       describe('and the define function is provided', function() {
         beforeEach(function() {
-          this.additionalParam = {};
+          this.additionalParam = [[1], 2];
           this.module = this.app.module('Mod', this.defineSpy, this.additionalParam);
         });
```

Comment:
what happens if you do want to pass multiple additional params - I can imagine some people might be using this. Can we add a test to see if it passes?